### PR TITLE
Accessibility Updates

### PR DESCRIPTION
**Pull request by ananyo2012**
Update Footer in CoC Page

### CODE REVIEW

File: coc.html [hunk 1]
```diff
@@ -146,12 +146,21 @@ <h2 class="text-center">
...
+                <a href="https://github.com/pythonindia"><i aria-label="GitHub" class="fa fa-github"></i></a>
+            </p>
+            <p><a href="mailto:contact@in.pycon.org">contact@in.pycon.org</a></p>
             <p> Copyright &copy; 2020 Python India.</a> </p>
             <p>Image Credits to Rakshith N for his shot of <a href="https://dribbble.com/shots/6084190-Bangalore-Palace">Bangalore Palace</a></p>
         </div>
 
         <div class="row col-md-12 wc-footer">
-            <p><a href="http://webchirpy.com" target="_blank" class="" /> Design with <img class="pulse2" src="images/heart.png" style="vertical-align:middle;margin-top:-5px;" /> by Webchirpy</a>
+            <p><a href="http://webchirpy.com" target="_blank" class="" /> Design with <img class="pulse2" alt="Love" src="images/heart.png" style="vertical-align:middle;margin-top:-5px;" /> by Webchirpy</a>
```
Comment (SaptakS) [inline coc.html]: Can you add `rel="noreferrer noopener"`?

File: index.html [hunk 2]
```diff
@@ -117,20 +117,20 @@ <h5 class="date">
...
+                <a href="https://github.com/pythonindia"><i aria-label="GitHub" class="fa fa-github"></i></a>
             </p>
             <p><a href="mailto:contact@in.pycon.org">contact@in.pycon.org</a></p>
             <p> Copyright &copy; 2020 Python India.</a> </p>
             <p>Image Credits to Rakshith N for his shot of <a href="https://dribbble.com/shots/6084190-Bangalore-Palace">Bangalore Palace</a></p>
         </div>
 
         <div class="row col-md-12 wc-footer">
-            <p><a href="http://webchirpy.com" target="_blank" class="" /> Design with <img class="pulse2" src="images/heart.png" style="vertical-align:middle;margin-top:-5px;" /> by Webchirpy</a>
+            <p><a href="http://webchirpy.com" target="_blank" class="" /> Design with <img class="pulse2" alt="Love" src="images/heart.png" style="vertical-align:middle;margin-top:-5px;" /> by Webchirpy</a>
```
Comment (SaptakS) [inline index.html]: Same as above.

**Comment by bhansa** [comment]
We can also use a small snippet which I created earlier for pyconf website, it adds noopener for all the links which are having `target="_blank"`.
https://github.com/HydPy/hydpyconf2019/blob/master/assets/js/main.js#L3

**Comment by SaptakS** [comment]
@bhansa umm that's good, but that works only when document is ready for DOM. On a slow internet, that's enough time for vulnerability. Also, I think we should go in a direction that if someone chooses to restrict using javascript from their browser, that doesn't deprive them of such security fixes. So even though this is a nice hack, I would really prefer to have the change in the HTML itself.

**Comment by SaptakS** [comment]
@ananyo2012 LGTM. There are some contrast issues in color going by WCAG AA standards but since this is not the final color scheme, I am happy to ignore them for now.

**Pull request merged**